5_core_acc_analysis/3_explore_exo_modules.py

```diff
@@ -158,14 +158,14 @@
 
 # Core genes highly co-expressed with both exoS and exoU
 exo_core_both = exo_core_corr[
-    (exo_core_corr["corr to exoS"] > 0.4) & (exo_core_corr["corr to exoU"] > 0.4)
+    (exo_core_corr["corr to exoS"] > 0.4) & (exo_core_corr["corr to exoU"] > 0.2)
```

Comment:
Is there a general reason you're having to tweak some of these cutoffs? Is this because of the decrease in signal strength you mentioned at the top (and by signal strength, do you just mean expression levels are overall lower in core genes)?

Reply:
The signal strength is more referring to the difference in magnitude of the "least stable" bars using the old data (top) vs the new data (bottom). The change in thresholds here was based on a manual inspection and making sure I captured the genes in the two clusters.

![image](https://user-images.githubusercontent.com/13107360/146815486-8db608bb-b4e8-402a-bde9-6e976606a206.png)

![image](https://user-images.githubusercontent.com/13107360/146815789-35d6a119-0661-4abb-a52e-784bc5b8dfd9.png)